src/main/java/net/minestom/server/network/packet/server/play/SoundEffectPacket.java

```diff
@@ -11,15 +12,24 @@
 public class SoundEffectPacket implements ServerPacket {
 
     public int soundId;
-    public SoundCategory soundCategory;
+    public int soundCategory;
```

Comment:
Same as earlier comment: don't replace an enum type to its integral representation.